### PR TITLE
Fix formatting HAProxy doc

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?
- Moved the agent IP note inline with example.
- Fixed formatting of bullets.
## What type of changes does your doc introduce?
- [x] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

## If you included a comment with your commit, it appears here:

### CODE REVIEW

**Comment by joel-hamill** [comment]
@nlsun let me know what you think.

File: 1.8/administration/haproxy-adminrouter.md [hunk 1]
```diff
@@ -10,21 +10,22 @@ The HTTP Proxy must perform on-the-fly HTTP request and response header modifica
...
+    ```
+    <taskname>.<framework_name>.agentip.dcos.<dcos_name>.directory
+    ```
+    
+    Where:
+    
+    * `taskname`: The name of the task
+    * `framework_name`: The name of the framework, if you are unsure, it is likely `marathon`
+    * `dcos_name`: The name of your DC/OS, which defaults to `thisdcos`
+
```
Comment (nlsun) [inline 1.8/administration/haproxy-adminrouter.md]: The github rendered markdown doesn't have a space between the last bullet  and the next code block, does the site render it better?

**Comment by nlsun** [comment]
Other than that it looks good

File: 1.8/administration/haproxy-adminrouter.md (hunk 1 above)
Comment (joel-hamill) [inline 1.8/administration/haproxy-adminrouter.md]: yeah, looks like this: https://cl.ly/3P3r2o3J2j3i

**Comment by joel-hamill** [comment]
@sascala ready for merge

File: 1.8/administration/haproxy-adminrouter.md (hunk 1 above)
Comment (sascala) [inline 1.8/administration/haproxy-adminrouter.md]: Suggest: periods at the end of the sentences above.
"The name of the framework (also called a DC/OS service). If you are unsure, it is likely `marathon`."
"The name of your DC/OS instance. Default is `thisdcos`."

**Pull request merged**